06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,183 @@
+const validFile = require('./books/validFile.json');
```

Comment:
Great job getting your test data from actual JSON files.

Can this file be named better?